pkg/kritis/apis/kritis/v1beta1/securitypolicy.go

```diff
@@ -36,6 +36,7 @@ type ImageSecurityPolicySpec struct {
 	ImageAllowlist                   []string                         `json:"imageAllowlist"`
 	PackageVulnerabilityRequirements PackageVulnerabilityRequirements `json:"packageVulnerabilityRequirements"`
 	AttestationAuthorityName         string                           `json:"attestationAuthorityName"`
```

Comment:
Let's make sure to also change this to Note name.  (Same in build policy.)  We can, and maybe should, do it in a separate PR, let's just not forget about it.

Reply:
Ack.